src/es/hopscotch.js

```diff
@@ -102,6 +102,19 @@ export default class Hopscotch {
   }
 
   /**
+   * Skips to a given step in the tour
+   *
+   * @param {Number} stepNumber - Zero-based step number
+   * @return {Hopscotch} - Returns itself to allow chaining method calls 
+   */
+  showStep(stepNumber) {
+    if (currentTour) {
```

Comment:
Should this, and (in retrospect) other step traversal methods, complain if there's no tour in progress?